acceptance/bundle/upload/internal_server_error/output.txt

```diff
@@ -0,0 +1,10 @@
+Uploading bundle files to /Workspace/Users/[USERNAME]/.bundle/test-bundle/default/files...
+Error: Internal Server Error (500)
+
+Endpoint: POST [DATABRICKS_URL]/api/2.0/workspace-files/import-file/Workspace%2FUsers%2F[USERNAME]%2F.bundle%2Ftest-bundle%2Fdefault%2Ffiles%2Ffile_to_upload.txt?overwrite=true
+HTTP Status: 500 Internal Server Error
+API error_code: 
```

Comment:
This is the whitespace line. We could write `<empty>` as placeholder if there is no value.